misc/testerina/modules/testerina-core/pom.xml

```diff
@@ -64,6 +68,23 @@
             <artifactId>slf4j-jdk14</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.swagger</groupId>
+            <artifactId>swagger-models</artifactId>
+            <version>2.0.0-rc2</version>
```

Comment:
We can't remove the version as it is not defined in parent poms. Since this is a test scope dependency, I didn't bother to have a property for the version either.

Reply:
Actually this is in the parent pom, but seemed it's a outdated version.

Reply:
yes.. Swagger connector uses an inline version and one of the latest versions.